packages/gulp-babili/__tests__/__snapshots__/gulp-babili-test.js.snap

```diff
@@ -1,11 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`gulp-babili comments should remove all comments when false 1`] = `"foo(),bar(),baz();"`;
+exports[`gulp-babili comments should remove all comments when false 1`] = `"function foo(){}bar();var a=baz();"`;
```

Comment:
Yes lol.. Dont even know how its working.. Just want to let you guys know there is something wrong!